src/docs/premium.mdx

```diff
@@ -10,7 +10,7 @@ custom_edit_url: https://github.com/EarthMC/earthmc-docs/blob/main/src/docs/prem
 
 [Here is the store page](https://store.earthmc.net/category/premium) if you are interested in buying premium. 
 
-Premium is a paid rank that helps fund the server and its development. It ensures thousands of players can play the server for free. At EarthMC we believe that players should not be able to pay for progress, something that sets us apart from most other large servers out there. Gold, the server’s currency, funds everything related to progress on the server. It’s therefore not possible to pay real money for gold. Instead, we offer convenience for those who chose to support the server through premium. Premium has many convenient features that could upgrade your EarthMC experience; 
+Premium is a paid rank that helps fund the server and its development. It ensures thousands of players can play the server for free. At EarthMC we believe that players should not have to pay for progress, something that sets us apart from most other large servers out there. Gold, the server’s currency, funds everything related to progress on the server. It’s therefore not possible to pay real money for gold. Instead, we offer convenience for those who chose to support the server through Premium. Premium has many convenient features that could upgrade your EarthMC experience; 

```

Comment:
Premium should not be capitalized here.

Reply:
IMO, the proposed corrections improve what we try to convey. 

"should not be able" fits better than "should not have to", whereas the capitalization in "Premium" is a mistake I made, as s foreign speaker.

Reply:
I think Premium should be capitalized. It's being used as the proper name of the subscription, not simply an adjective.